demo/some-plugin.js

```diff
@@ -0,0 +1,69 @@
+const { BasePlugin, registerPlugin } = KalturaPlayer;
+
+import {
+  AnotherIconComponent,
+  AnotherPanelItemComponent,
+  IconComponent,
+  PanelItemComponent,
+  SomeIconComponent, SomePanelItemComponent
+} from './components.js';
+
+export const pluginName = 'somePlugin';
+
+export class somePlugin extends BasePlugin {
+  static defaultConfig = {};
+
+  constructor(name, player) {
+    super(name, player);
+  }
+
+  loadMedia() {
+    const panelItemAId = this.player.getService('sidePanelsManager').addItem({
+      label: 'A',
+      expandMode: 'alongside',
+      renderIcon: IconComponent,
+      position: 'left',
+      presets: ['Playback', 'Live'],
+      renderContent: PanelItemComponent,
+    });
+
+    const PanelItemBId = this.player.getService('sidePanelsManager').addItem({
+      label: 'B',
+      expandMode: 'alongside',
+      renderIcon: AnotherIconComponent,
+      position: 'left',
+      presets: ['Playback', 'Live'],
+      renderContent: AnotherPanelItemComponent,
+    });
+
+    const PanelItemCId = this.player.getService('sidePanelsManager').addItem({
+      label: 'B',
+      expandMode: 'alongside',
+      renderIcon: SomeIconComponent,
+      position: 'bottom',
+      presets: ['Playback', 'Live'],
+      renderContent: SomePanelItemComponent,
+    });
+
+    this.ready.then(() => {
```

Comment:
I would add another one to right to see the switching between left and right panels

Reply:
done